src/components/slider/slider.tsx

```diff
@@ -179,10 +194,10 @@ export class Slider implements LabelableComponent, FormComponent, InteractiveCom
 
   render(): VNode {
     const id = this.el.id || this.guid;
+    const maxProp = Array.isArray(this.value) ? "maxValue" : "value";
+    const value = Array.isArray(this.value) ? this.maxValue : this.value;
     const min = this.minValue || this.min;
-    const max = this.maxValue || this.value;
-    const maxProp = this.isRange ? "maxValue" : "value";
-    const value = this[maxProp];
+    const max = this.maxValue || value;
```

Comment:
With the recent changes, isn't `max` === `value` here? 

```tsx
const value = 
  Array.isArray(this.value) ? 
    this.maxValue : // if we have a range, `value` will be `this.maxValue` (a)
    this.value; // else it's `this.value` (b)

// if we have a range, `this.maxValue` will be defined and assigned to `value` already (a)
// if not, it will be `this.value` (b)
const max = this.maxValue || value;
```

Reply:
I think max is still used when its not a range. I don't think the logic changed right?

`const max = this.maxValue || this.value;`

Reply:
Correct, the logic didn't change, but the assignments could be simplified since I think all of these are equivalent now:

```ts
const max = Array.isArray(this.value)? value[1]: value;
const max = this.maxValue || this.value;
const max = value;
```